app/views/website_members/index.html.haml

```diff
@@ -18,9 +19,13 @@
             %th.text-right Actions
         %tbody.websites
           - @website_members.each do |website_member|
-            %tr
-              %td= website_member.user.name
-              %td= website_member.user.email
-              %td= website_member.role
-              %td.text-right
-                = link_to ' Delete', "/website_members/#{website_member.id}", :method => :delete, data: {confirm: "Are you sure?"}, :class => "delete_website btn btn-danger pull-right fa fa-close"
\ No newline at end of file
+            - unless @current_website_member == website_member
+              = simple_form_for website_member, wrapper: :vertical_form do |f|
+                %tr
+                  %input{:name => "id", :type => "hidden", :value => "#{website_member.id}"}/
+                  %td= website_member.user.name
+                  %td= website_member.user.email
+                  %td
+                    = select_tag 'website_member_role[role]', options_for_select([['Owner', 1], ['User', 2]], selected: website_member.role.value), id: nil, style: 'max-width: 90px;', data: { remote: true, url: change_role_website_member_path(website_member), method: 'put', params: '_method=put', type: 'json' }
+                  %td.text-right
+                    = link_to ' Delete', "/website_members/#{website_member.id}", :method => :delete, data: {confirm: "Are you sure?"}, :class => "delete_website btn btn-danger pull-right fa fa-close"
```

Comment:
use for url here `website_member_url(website_member)` instead of building it by hand